io/pci/pci_hotplug.py

```diff
@@ -42,28 +42,35 @@ def setUp(self):
         """
         cmd = "uname -p"
         if 'ppc' not in process.system_output(cmd, ignore_status=True):
-            self.cancel("Processor is not ppc64")
-        if cpu._list_matches(cpu._get_cpu_info(), 'pSeries'):
+            self.skip("Processor is not ppc64")
+        cmd = "cat /proc/cpuinfo"
+        if 'pSeries' in process.system_output(cmd, ignore_status=True):
```

Comment:
@abdhaleegit That was his exact intention here. To differentiate PowerVM and PowerNV